.github/workflows/downstream.yml

```diff
@@ -0,0 +1,50 @@
+name: IntegrationTest
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  test:
+    name: ${{ matrix.package.repo }}/${{ matrix.package.group }}
+    runs-on: ${{ matrix.os }}
+    env:
+      GROUP: ${{ matrix.package.group }}
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version: [1]
+        os: [ubuntu-latest]
+        package:
+          - {user: SciML, repo: SBMLToolkit.jl, group: All}
```

Comment:
is there a straightforward way to extend this for multiple packages? Ideally, can we have LCSB-BioCore/COBREXA.jl  too?

Reply:
yes. see https://github.com/SciML/ModelingToolkit.jl/blob/master/.github/workflows/Downstream.yml#L20-L27

Reply:
ill add you guys now